Mathlib/Analysis/LocallyConvex/AbsConvex.lean

```diff
@@ -40,6 +42,99 @@ open NNReal Pointwise Topology
 
 variable {𝕜 E F G ι : Type*}
 
+section AbsolutelyConvex
+
+variable (𝕜) [SeminormedRing 𝕜] [SMul 𝕜 E] [SMul ℝ E] [AddCommMonoid E]
+/-- A set is absolutely convex if it is balanced and convex. -/
+def AbsConvex (s : Set E) : Prop := Balanced 𝕜 s ∧ Convex ℝ s
+
+variable {𝕜}
+
+theorem AbsConvex.empty : AbsConvex 𝕜 (∅ : Set E) := ⟨balanced_empty, convex_empty⟩
+
+theorem AbsConvex.univ : AbsConvex 𝕜 (Set.univ : Set E) := ⟨balanced_univ, convex_univ⟩
```

Comment:
```suggestion
theorem AbsConvex.univ : AbsConvex 𝕜 (.univ : Set E) := ⟨balanced_univ, convex_univ⟩
```
if you want

Reply:
Presumably `univ` (without the `.`) also okay?